### DISCUSSION TITLE
How to make a character float

### DISCUSSION THREAD

**Discussion by GummyRon** [CodenameCrew/CodenameEngine#483] [Q&A]
im needing of a character script that i can use to make a character float

example: 3D Dave

the reason to why im needing this is because im not that skilled at coding with haxe and this is what happens if i try to make one (gif below)
![badscriptexample](https://github.com/user-attachments/assets/e906b8fc-9d55-4fcf-a170-dd8a3eaff783)

**Comment by TeriosSonic** [comment]
you should probably ask around in the #modding-help channel in the official CNE Discord server, this is not the place for this kind of stuff

**Reply by GummyRon** [reply]
oh my bad
